test/testcafe/spec/UnlockAccountChallenge_gen3_spec.js

```diff
@@ -0,0 +1,266 @@
+import TerminalPageObject from '../framework/page-objects/TerminalPageObject';
+import smartProbingRequired from '../../../playground/mocks/data/idp/idx/smart-probing-required';
+import launchAuthenticatorOption from '../../../playground/mocks/data/idp/idx/identify-with-device-launch-authenticator';
+
+
+const identifyLockedUserMock = RequestMock()
+  .onRequestTo('http://localhost:3000/idp/idx/introspect')
+  .respond(xhrIdentifyWithUnlock)
+  .onRequestTo('http://localhost:3000/idp/idx/unlock-account')
+  .respond(xhrUserUnlockAuthSelector)
+  .onRequestTo('http://localhost:3000/idp/idx/challenge')
+  .respond(xhrUserUnlockEmailChallenge)
+  .onRequestTo('http://localhost:3000/idp/idx/challenge/answer')
+  .respond(xhrUserUnlockSuccess);
+
+const identifyMock = RequestMock()
+  .onRequestTo('http://localhost:3000/idp/idx/introspect')
+  .respond(xhrIdentifyWithPassword);
+
+const errorUnlockAccount = RequestMock()
+  .onRequestTo('http://localhost:3000/idp/idx/introspect')
+  .respond(xhrIdentifyWithUnlock)
+  .onRequestTo('http://localhost:3000/idp/idx/unlock-account')
+  .respond(xhrUserUnlockAuthSelector)
+  .onRequestTo('http://localhost:3000/idp/idx/challenge')
+  .respond(xhrUserUnlockEmailChallenge)
+  .onRequestTo('http://localhost:3000/idp/idx/challenge/answer')
+  .respond(xhrErrorUnlockAccount);
+
+const identifyLockedUserLandOnAppMock = RequestMock()
+  .onRequestTo('http://localhost:3000/idp/idx/introspect')
+  .respond(xhrIdentifyWithUnlock)
+  .onRequestTo('http://localhost:3000/idp/idx/unlock-account')
+  .respond(xhrUserUnlockAuthSelector)
+  .onRequestTo('http://localhost:3000/idp/idx/challenge')
+  .respond(xhrUserUnlockEmailChallenge)
+  .onRequestTo('http://localhost:3000/idp/idx/challenge/answer')
+  .respond(xhrUserUnlockSuccessLandOnApp);
+
+const xhrUserUnlockAuthSelectorWithOneAuthenticator = JSON.parse(JSON.stringify(xhrUserUnlockAuthSelector));
+xhrUserUnlockAuthSelectorWithOneAuthenticator.remediation.value[0].value[1].options.splice(-1, 1);
+
+const identifyLockedUserMockWithOneAuthenticator = RequestMock()
+  .onRequestTo('http://localhost:3000/idp/idx/introspect')
+  .respond(xhrIdentifyWithUnlock)
+  .onRequestTo('http://localhost:3000/idp/idx/unlock-account')
+  .respond(xhrUserUnlockAuthSelectorWithOneAuthenticator)
+  .onRequestTo('http://localhost:3000/idp/idx/challenge')
+  .respond(xhrUserUnlockEmailChallenge)
+  .onRequestTo('http://localhost:3000/idp/idx/challenge/answer')
+  .respond(xhrUserUnlockSuccess);
+
+const signInDeviceMock = RequestMock()
+  .onRequestTo('http://localhost:3000/idp/idx/introspect')
+  .respond(smartProbingRequired)
+  .onRequestTo('http://localhost:3000/idp/idx/authenticators/okta-verify/launch')
+  .respond(launchAuthenticatorOption);
+
+
+const rerenderWidget = ClientFunction((settings) => {
+  window.renderPlaygroundWidget(settings);
+});
+
+fixture('Unlock Account').meta('gen2', false);
```

Comment:
nit: (want the spec name to differ from the Gen2 version 
```suggestion
fixture('Unlock Account - Gen 3').meta('gen2', false);
```